README.md

```diff
@@ -42,15 +40,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: @magorsy26
 
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: Kindly make sure you use your social link with username to make your README documentation professional. e.g:
```
GitHub: [@magorsy26](https://github.com/magorsy26)
```